process/testing/util/delete-az-rg.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
I can't see if this file is executable or not in the PR, but don't forget to `chmod +x` before committing it